### PR TITLE
Refactor common data capture

### PR DESCRIPTION
**Pull request by zhexuany**

### CODE REVIEW

File: docs/2-get-started/4-add-devices-to-org.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 3
+---
+
+## 在您的组织中添加设备
+### 为您的设备安装刻行的端侧软件
+进入【组织管理】-【设备】-【添加设备】，获取设备 Agent 安装命令
+![device-command](./img/common-task-2.png)
```
Comment (yujingz) [inline docs/2-get-started/4-add-devices-to-org.md:8]: no such image

**Comment by yujingz** [comment]
有几个素材好像没传上来

File: docs/2-get-started/4-add-devices-to-org.md (hunk 1 above)
Comment (zhexuany) [reply docs/2-get-started/4-add-devices-to-org.md:8]: Resolved. But some images are duplicated.

File: docs/2-get-started/4-add-devices-to-org.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+### 将设备添加至项目
+
+进入对应的项目，选择【项目设备】-【添加设备】-【从组织设备选择】，选择添加刚才安装的设备到当前项目。
+![project-devices](./img/project-devices.png)
+
+## 配置采集规则
+
+进入【组织管理】-【设备】-【编辑数采规则】，配置整体的数采规则信息。
+![device-collector](./img/common-task-1.png)
```
Comment (yujingz) [inline docs/2-get-started/4-add-devices-to-org.md:27]: Think we are still missing this one.

**Comment by yujingz** [comment]
Yeah there are assets that we dont use anymore. We should clean them up some time. But it does not affect project performance so it doesn't bother me much

**Comment by yujingz** [comment]
目前这里面的内容和我们实际产品已经有比较大的区别了，我尝试改了下还是太多不一样了，暂时就不合入了，更新后可以重新 review

**Pull request closed without merging**